keyboards/lets_split/keymaps/poker/keymap.c

```diff
@@ -155,77 +155,30 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
 };
 
-#ifdef AUDIO_ENABLE
-float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
-float tone_dvorak[][2]     = SONG(DVORAK_SOUND);
-float tone_colemak[][2]    = SONG(COLEMAK_SOUND);
-#endif
+layer_state_t layer_state_set_user(layer_state_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
```

Comment:
The [tri-layer feature](https://docs.qmk.fm/#/feature_tri_layer) should be used here instead.
```suggestion
```

Reply:
I copied this from the [default Planck keymap](https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/keymaps/default/keymap.c#L181-L183). I'll take a look the documentation you linked.

Reply:
Updated to use Tri Layer feature.